frontend/src/components/regions/UserActions/ActivityDashboardBarGraph.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+// import { cardColors } from "../../../colors";
+
+const data = [
```

Comment:
Lets try and use some data from the API endpoint example. See if that works

Reply:
I used some data from the example api(and left some out). I commented out the YAxis

Reply:
The eventTypeColor names might seem a bit strange.But I used them to match the colors in the brief

Reply:
The colors seeming strange should have been a massive red flag for you.  The colours exactly match the event types we need to represent here